.github/workflows/cleanup-old.yml

```diff
@@ -12,11 +12,11 @@ jobs:
     name: Remove old nightly releases
     runs-on: ubuntu-latest
     steps:
-      - name: Remove all but last 10 releases
+      - name: Remove all but last 90 releases
         env:
           GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
         shell: bash
         run: |
-          RELEASES=`gh release list --repo dsp-testing/codeql-cli-nightlies | grep "Pre-release" | sed '1,10d' | cut -f 3`
+          RELEASES=`gh api --paginate repos/dsp-testing/codeql-cli-nightlies/releases --jq 'map(select(.prerelease == true)).[90:].[] | .tag_name'`
```

Comment:
Worth a comment. The API returns results in reverse chronological order, so `[0:89]` are the last 90 releases, and `[90:]` are older than that.